apps/mull-ui/src/app/app.tsx

```diff
@@ -41,6 +44,15 @@ export const App = () => {
             <EventPage event={dummyEvent} prevPage={'Review'} />
           </div>
         </Route>
+        {/* Without this outter Route component, the inner components would show on all routes that are not matched */}
+        <Route exact path={[`${ROUTES.DISCOVER}`, `${ROUTES.UPCOMING}`, `${ROUTES.MYEVENTS}`]}>
+          <SubNavigationBar />
+          <SwipeableRoutes>
+            <Route path={ROUTES.DISCOVER} component={() => <div>DISCOVER!</div>} />
+            <Route path={ROUTES.UPCOMING} component={() => <div>UPCOMING!</div>} />
+            <Route path={ROUTES.MYEVENTS} component={() => <div>MYEVENTS!</div>} />
+          </SwipeableRoutes>
+        </Route>
```

Comment:
One issue im seeing right now with styling is that the Subnavbar is set to be in the page-container, which is then pushing the actual page content further down than it should.

![image](https://user-images.githubusercontent.com/23544999/100519347-5015dc80-3165-11eb-8845-e3c93d157709.png)

I propose this solution: Remove the page-container class from SubNavigationBar, and put page-container on a container div. So something like this:
```suggestion
        <Route exact path={[ROUTES.DISCOVER, ROUTES.UPCOMING, ROUTES.MYEVENTS]}>
          <div className="page-container">
            <SubNavigationBar />
            <SwipeableRoutes>
              <Route path={ROUTES.DISCOVER} component={() => <div>DISCOVER!</div>} />
              <Route path={ROUTES.UPCOMING} component={() => <div>UPCOMING!</div>} />
              <Route path={ROUTES.MYEVENTS} component={() => <div>MYEVENTS!</div>} />
            </SwipeableRoutes>
          </div>
        </Route>
```